### ISSUE TITLE
请问下为什么表格分析训练了十多个小时acc一直为0，epoch: [1/50]一直没变 

### ISSUE THREAD

**Issue by 965662766** [PaddlePaddle/PaddleOCR#3723]
![image](https://user-images.githubusercontent.com/85744622/129820893-07bf24be-3811-4087-8ecb-ac8f3b5f02ac.png)
 
其中配置信息

Global:
  use_gpu: true
  epoch_num: 50
  log_smooth_window: 20
  print_batch_step: 5
  save_model_dir: ./output/table_r50/
  save_epoch_step: 5
  # evaluation is run every 400 iterations after the 0th iteration
  eval_batch_step: [0, 400]
  cal_metric_during_train: True
  pretrained_model:
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_words/ch/word_1.jpg
  # for data or label process
  character_dict_path: ppocr/utils/dict/table_structure_dict.txt
  character_type: en
  max_text_length: 100
  max_elem_length: 500
  max_cell_num: 500
  infer_mode: False
  process_total_num: 0
  process_cut_num: 0

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  clip_norm: 5.0
  lr:
    learning_rate: 0.0001
  regularizer:
    name: 'L2'
    factor: 0.00000

Architecture:
  model_type: table
  algorithm: TableAttn
  Backbone:
    name: ResNet
    layers: 50
  Head:
    name: TableAttentionHead
    hidden_size: 256
    l2_decay: 0.00001
    loc_type: 2

Loss:
  name: TableAttentionLoss
  structure_weight: 100.0
  loc_weight: 10000.0

PostProcess:
  name: TableLabelDecode

Metric:
  name: TableMetric
  main_indicator: acc

Train:
  dataset:
    name: PubTabDataSet
    data_dir: /data2/project/dataset/pubtabnet/train/
    label_file_path: /data2/project/dataset/pubtabnet/PubTabNet_train_2.0.0.jsonl
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - ResizeTableImage:
          max_len: 1024
      - TableLabelEncode:
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - PaddingTableImage:
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'structure', 'bbox_list', 'sp_tokens', 'bbox_list_mask']
  loader:
    shuffle: True
    batch_size_per_card: 2
    drop_last: True
    num_workers: 1

Eval:
  dataset:
    name: PubTabDataSet
    data_dir: /data2/project/dataset/pubtabnet/val/
    label_file_path: /data2/project/dataset/pubtabnet/PubTabNet_val_min_2.0.0.jsonl
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - ResizeTableImage:
          max_len: 1024
      - TableLabelEncode:
      - NormalizeImage:
          scale: 1./255.
          mean: [0.485, 0.456, 0.406]
          std: [0.229, 0.224, 0.225]
          order: 'hwc'
      - PaddingTableImage:
      - ToCHWImage:
      - KeepKeys:
          keep_keys: ['image', 'structure', 'bbox_list', 'sp_tokens', 'bbox_list_mask']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1
    num_workers: 1

**Comment by MissPenguin** [comment]
先不要改默认参数训练试试呢

**Comment by 965662766** [comment]
参数都不改的情况下是acc是能上去的，但我表格尺寸很大，默认的方式出来的结果效果不理想，所以想更改Backbone 和max_len 来训练试下看效果，或者有没有其它方式可以解决呢？

**Comment by cpwan** [comment]
Similar problem here. After 400 epochs, my loc loss is about 2.9 but my structure loss is 250. I used the config from #4497. The model does not return any bounding box and the structure part is flooded with `<thead>` token.

**Comment by nissansz** [comment]
怎么改成resnet backbone？
